resources/web/wwi/server.js

```diff
@@ -3,21 +3,29 @@
 
 class Server { // eslint-disable-line no-unused-vars
   constructor(url, view, onready) {
+    this.url = url;
     this.view = view;
     this.onready = onready;
-
-    // url has the following form: "ws(s)://cyberbotics1.epfl.ch:80/simple/worlds/simple.wbt"
-    var n = url.indexOf('/', 6);
-    var m = url.lastIndexOf('/');
-    this.url = 'http' + url.substring(2, n); // e.g., "http(s)://cyberbotics1.epfl.ch:80"
-    this.project = url.substring(n + 1, m - 7); // e.g., "simple"
-    this.worldFile = url.substring(m + 1); // e.g., "simple.wbt"
+    // url has one of the following form:
+    // "ws(s)://cyberbotics1.epfl.ch:80/simple/worlds/simple.wbt", or
+    // "wss://cyberbotics1.epfl.ch/1999/session
+    //  ?url=webots://github.com/cyberbotics/webots/branch/master/projects/languages/python"
```

Comment:
It seems that the URL should link to a world file:
```suggestion
    //  ?url=webots://github.com/cyberbotics/webots/branch/master/projects/languages/python/worlds/example.wbt"
```